lib/fetch/sanitize.js

```diff
@@ -124,3 +124,23 @@ export function sanitizeTicket(json) {
     available: get('ticket.available'),
   };
 }
+
+export function sanitizeBadger(json) {
+  const get = makeGetter(json);
+  return {
+    id: json.id,
+    firstName: get('badger.first-name'),
+    lastName: get('badger.last-name'),
+    jobTitle: get('badger.job-title'),
+    imageUrl: get('badger.image-url'),
+    startDate: get('badger.start-date'),
+    about: get('badger.about'),
+    skills: get('badger.skills'),
+    influence: get('badger.influence'),
```

Comment:
@asavin me and @grahammendick  were just checking and null fields don't cause any issue on this stage. It will fetch null and add a field with a null result on the state